src/gdbstub_impl/ext/base.rs

```diff
@@ -143,6 +147,30 @@ impl<T: Target, C: Connection> GdbStubImpl<T, C> {
                 }
                 HandlerStatus::Handled
             }
+            Base::qXferMemoryMapRead(cmd) => {
```

Comment:
You'll have to move this hander out into a separate `ext` mod.

(meta): once this PR is merged, I'll pop-open a tracking task for myself to de-dupe the `qXfer` logic between the two packets. some copy-paste is fine for now, but I know that there'll be more of these packets coming down the line, and there'll need to be some way to keep the code DRY.